### ISSUE TITLE
Schedule for runs

### ISSUE THREAD

**Issue by mxxcon** [dabutvin/ImgBot#32]
Please consider somehow adding an ability to schedule compression runs.
For example run it only once a day/week/month/whenever. That way active repos where images often change don't get flooded with commits from this bot.

**Comment by dabutvin** [comment]
This would be a great addition when we support a config file https://github.com/dabutvin/ImgBot/issues/2

Thanks for the issue!

**Comment by dabutvin** [comment]
I think offering daily/weekly/monthly as options is a good place to start.

In an effort to always keep ImgBot free by keeping costs super low I want to make sure not to clone repos on a schedule that have no activity.

One thing you can do in the meantime is to not delete the `imgbot` branch until you are ready for more image optimizations. We won't make a new PR if the old branch is hanging around.
